cognee-frontend/src/modules/chat/hooks/useChat.ts

```diff
@@ -14,7 +14,7 @@ const fetchMessages = () => {
     .then(response => response.json());
 };
 
-const sendMessage = (message: string, searchType: string) => {
+const sendMessage = (message: string, searchType: string, topK: number = 10) => {
   return fetch("/v1/search/", {
```

Comment:
nit: remove default value here, since we're already defining it in SearchView component